stylesheets/blue/components/_button_primary.scss

```diff
@@ -4,6 +4,7 @@
 //
 // default - Normal state
 // .ButtonPrimary--fullWidth - A button that takes the whole width of its parent
+// .ButtonPrimary--brightTurquoise - A button that uses the brightTurquoise color
```

Comment:
It works like an alternative to the yellow button. I like the name, it conveys the purpose of its use.